### PR TITLE
Add flashSuitChecked to transitioning with stored fall speed

### PR DESCRIPTION
**Pull request by kjbranch**
Didn't add it to strats with screw to break a block while still having stored fall speed, because I wasn't able to test it.

### CODE REVIEW

**Review by osse101 — DISMISSED** [review]
Moonfalling with Screw Attack let me keep the flash suit.  There's nothing about it that would trigger the flash suit other than accidently buffering a jump, but holding angle and jump prevent that from happening.

The did the dachora moondance with a flash suit then broke the blocks to reach the right side with Screw.  The only thing I had to do differently was bombjumping to reach the top of the bomb wall since you cannot jump morph.  Croc speedway and Bowling moondances won't work because of the Crystal Flash, but a Beetom moondance should be fine as long as you can carry the beetom to the door you need it for without losing the flash suit.

When using the stored moonfall with 1 tile of stored speed you need to be careful about buffering the use of the flash suit, by releasing jump asap.  I tested this with Gauntlet Etank right->left stored moonfall+flash suit to cross the room.

File: region/norfair/east/Frog Speedway.json [hunk 1]
```diff
@@ -725,6 +729,7 @@
         {"types": ["missiles", "super"], "requires": []},
         {"types": ["powerbomb"], "requires": ["never"]}
       ],
+      "flashSuitChecked": true,
```
Comment (kjbranch) [inline region/norfair/east/Frog Speedway.json:732]: Added flashSuitChecked to the strats that transition and use screw. I figure the setups and uses can be added later. I could attempt to work on it next, or start to do the easier strats in rooms. 

One thing I noticed is in this strat (speedless speedway) it says you can transition with stored fall speed and deal with the Beetoms with bombs/PBs - I'm pretty sure that youll lose the stored fall speed?

Comment (osse101) [reply region/norfair/east/Frog Speedway.json:732]: Correct, you cannot use bombs or Pbs.  And Screw depends on your amount of stored fall speed.

Comment (kjbranch) [reply region/norfair/east/Frog Speedway.json:724]: I removed the bomb/PB options. I wasn't sure how to update the screw options if they needed it

File: region/norfair/east/Frog Speedway.json [hunk 2]
```diff
@@ -700,14 +704,6 @@
           {"ammo": {"type": "Missile", "count": 1}},
           {"ammo": {"type": "Super", "count": 1}},
           "ScrewAttack",
```
Comment (osse101) [inline region/norfair/east/Frog Speedway.json:706]:
```suggestion
```

File: region/norfair/east/Frog Speedway.json [hunk 3]
```diff
@@ -756,14 +753,6 @@
...
```
Comment (osse101) [inline region/norfair/east/Frog Speedway.json:755]:
```suggestion
```

Comment (osse101) [reply region/norfair/east/Frog Speedway.json:755]: Tried it, and Screw doesn't kill them at all when you do the moonfall.

**Pull request merged**